spec/models/allocation_spec.rb

```diff
@@ -126,12 +126,6 @@
       expect(allocation.moves_count).to eq(0)
     end
 
-    it 'changes the status of all associated moves to cancelled' do
```

Comment:
just to double check that there are tests for this at the new endpoint to ensure move/allocation mirror the cancellation reason/comment etc?

Reply:
Yep, covered here: https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/spec/requests/api/v1/allocation_events_controller_cancel_spec.rb#L63
and
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/spec/services/event_log/move_runner_spec.rb#L87